demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -42,10 +42,86 @@ def contribution_converter(request: FixtureRequest, tmp_path_factory: TempPathFa
     return create_contribution_converter(request, tmp_path_factory)
 
 
+@patch('demisto_sdk.commands.split_yml.extractor.get_python_version')
+@patch('demisto_sdk.commands.init.contribution_converter.get_content_path')
+def test_convert_contribution_zip_updated_pack(get_content_path_mock, get_python_version_mock, tmp_path):
```

Comment:
It's odd to me that you are testing the update flow of the contribution conversion (aka when the contribution is an update to an existing pack) but you don't have existing pack files in the directory structure for the pack that we are simulating updating.

Reply:
We are using the pack from the testSuite. We have a default integration there, which has an empty readme, and in this flow, it stays empty.
If you prefer, I can use one of the existing packs from the `tests` folder.

Reply:
But I don't see anywhere in the test that we instantiate a pack with an integration that matches what the `ContributionConverter` will unpack and extract to package format. For instance, in line 85 we instantiate a `TestSuite` `Repo` object, but to instantiate a pack with the default integration I believe you would need to invoke `repo.setup_one_pack()` after that - can look [here](https://github.com/demisto/demisto-sdk/blob/706dfad2701d61746b5ce4847a6028d034ca428a/TestSuite/repo.py#L73) for the details.